package.json

```diff
@@ -6,8 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack --progress",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook --loglevel verbose",
+    "dev-storybook": "storybook dev -p 6007",
+    "storybook": "storybook dev -p 6007",
```

Comment:
`yarn storybook` currently runs exactly the same job as `yarn dev-storybook`, I guess it has to be modified or removed

Reply:
I've removed the extra dev command and set back to 6006.

It was there temporarily for me to get around some repeating terminal steps that would always popup, due to running multiple storybook projects at a time on my local.
This can be closed